tooling/add_stlink_rules.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
```

Comment:
Can you add a comment here (or maybe add to readme) about what this file does/when it should be run?

Reply:
I honestly have no idea why these are here